src/providers/analytics/analytics.model.ts

```diff
@@ -0,0 +1,48 @@
+export interface IAnalyticsProvider {
+    setCurrentPage(name: string);
+  
+    logEvent(category: string, event: string, params?: any);
+  
+    logError(message: string);
+  
+    setUserId(userId: string);
+  }
+  
+  export enum AnalyticsScreenNames {
+    CONTACT_DETAILS = 'contact details page', // this may need removing as could be candidate details now
+    FAIL_RESULTS_DEBRIEF = 'fail results debrief page',
+    HELP_CANDIDATE_DETAILS = 'help candidate details page',
+    HELP_DEBRIEF = 'help debrief page',
+    HELP_FINALISATION = 'help finalisation page',
+    HELP_GUIDE = 'help guide page',
+    HELP_HEALTH_DECLARATION = 'help health declaration page',
+    HELP_JOURNAL = 'help journal page',
+    HELP_TEST_REPORT = 'help test report page',
+    HELP_WAITING_ROOM = 'help waiting room',
+    HELP_WAITING_ROOM_TO_CAR = 'help waiting room to car page',
```

Comment:
Have you confirmed that the help pages are staying? I thought they were just for familiarisation.

Reply:
I've removed the HELP page related text. Wasn't sure if help pages were going to be added in the future, but if they are we can always add these back in.